test/functional/Valhalla/src/org/openj9/test/valhalla/NestAttributeTest.java

```diff
@@ -398,4 +398,42 @@ static public void testGetNestMembersAPINestMemberWrongPackage() throws Throwabl
 			} catch (IncompatibleClassChangeError e) { }
```

Comment:
This test should be moved out of the valhalla group and into the jdk11 set.

Reply:
https://github.com/eclipse/openj9/issues/2822